src/scss/base/_typography.scss

```diff
@@ -226,7 +226,7 @@ dfn {
 
 // Addresses styling not present in IE6/7/8/9
```

Comment:
Do we need some these styles now?

Reply:
I'll take a look and see if we can remove some of these